iothub/service/src/Amqp/AmqpConnectionHandler.cs

```diff
@@ -41,6 +41,12 @@ internal class AmqpConnectionHandler : IDisposable
 
         // The current delivery tag. Increments after each send operation to give a unique value.
         private int _sendingDeliveryTag;
+        
+        /// <summary>
+        /// Creates an instance of this class. Provided for unit testing purposes only.
+        /// </summary>
+        protected AmqpConnectionHandler()
```

Comment:
If for unit testing then protected internal, too.
```suggestion
        protected internal AmqpConnectionHandler()
```